GVRf/Framework/jni/engine/renderer/renderer.cpp

```diff
@@ -259,12 +258,30 @@ void Renderer::frustum_cull(Scene* scene,
 
         // Only push those scene objects that are inside of the frustum
         if (!is_inside) {
-            (*it)->set_in_frustum(false);
+            scene_object->set_in_frustum(false);
+            continue;
+        }
+
```

Comment:
A reasonable request, but we will also need the camera distance for sorting transparent object back-to-front and opaque objects front-to-back.